docs/activemq.md

```diff
@@ -0,0 +1,74 @@
+# `activemq` Metrics Receiver
+
+The `activemq` metrics receiver will be collected through the [Otel JMX Metric Gatherer](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/jmx-metrics).
+.
+
+## Prerequisites
+
+JMX support must be enabled in the [broker configuration](https://activemq.apache.org/jmx).
+
+## Configuration
+
+| Field                 | Default            | Description |
+| ---                   |--------------------| ---         |
+| `endpoint`            | `localhost:1099`   | Must be in the form of `host:port`.|
+| `username`            | not set by default | The configured username if JMX is configured to require authentication. |
+| `password`            | not set by default | The configured password if JMX is configured to require authentication. |
+| `collection_interval` | `60s`              | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
```

Comment:
Also `wildcard_refresh_interval`?

e.g.

```
| `wildcard_refresh_interval` | `60s` | The interval at which wildcard file paths in `include_paths` are refreshed. Specified as a time interval parsable by [time.ParseDuration](https://pkg.go.dev/time#ParseDuration). Must be a multiple of 1s.|
```

Reply:
I believe that would be covered by the logs receiver config, which will be implemented after this PR, but for now, this is simply for the metrics receiver config

Reply:
Oops. My mistake. Ignore this.